### ISSUE TITLE
Create Proposals versus Issues file

### ISSUE THREAD

**Issue by theresalech** [smartdevicelink/sdl_evolution#106]
Add attached information to sdl_evolution, and link to file from README
[SDL Evolution Proposal vs. Issue Reference Doc.docx](https://github.com/smartdevicelink/sdl_evolution/files/831997/SDL.Evolution.Proposal.vs.Issue.Reference.Doc.docx)

**Comment by joeljfischer** [comment]
Can you make that a PDF instead for those without Word?

**Comment by joeygrover** [comment]
Should just be a markdown file.

**Comment by kshala-ford** [comment]
There seems to be an issue with the tables in that markdown file. It's missing an empty line between the **Example** title and the table. 

```
####Examples
| SDL Platform     | Proposal   ....
```

change to 

```
####Examples

| SDL Platform     | Proposal   
```

to get the tables to work.

**Comment by theresalech** [comment]
Closed via PR #122
